src/PureScript/CST/Parser.purs

```diff
@@ -511,7 +511,7 @@ parseKindedVar open = do
 
 parseTypeParen :: SourceToken -> Parser (Recovered Type)
 parseTypeParen open = do
-  value <- parseType1
+  value <- parseType
```

Comment:
This is for #17 . I tried it out locally and it would now parse. It looks like this syntax does not appear in the package set.